README.md

```diff
@@ -1,37 +1,27 @@
-
 <p align="center">
-  <a href="https://boardgame.io/">
-    <img src="https://raw.githubusercontent.com/boardgameio/boardgame.io/main/docs/logo-optimized.svg?sanitize=true" alt="boardgame.io" />
-  </a>
+<a href="https://www.npmjs.com/package/@think-and-dev/cartesi-boardgame"><img src="https://img.shields.io/npm/v/@think-and-dev/cartesi-boardgame" alt="npm version" /></a>
 </p>
 
 <p align="center">
-<a href="https://www.npmjs.com/package/boardgame.io"><img src="https://badge.fury.io/js/boardgame.io.svg" alt="npm version" /></a>
-<a href="https://github.com/boardgameio/boardgame.io/actions?query=workflow%3ATests"> <img src="https://github.com/boardgameio/boardgame.io/workflows/Tests/badge.svg" alt='Build Status'></a>
-<a href='https://coveralls.io/github/boardgameio/boardgame.io?branch=main'><img src='https://coveralls.io/repos/github/boardgameio/boardgame.io/badge.svg?branch=main' alt='Coverage Status' /></a>
-<a href="https://gitter.im/boardgame-io"><img src="https://badges.gitter.im/boardgame-io.svg" alt="Gitter" /></a>
+  <strong>boardgame.io</strong> is an engine for creating turn-based games using JavaScript.
 </p>
 
-<p align="center">
-  <strong><a href="https://boardgame.io/documentation/#/">Read the Documentation</a></strong>
-</p>
+## What is T&D’s Boardgame Framework ?
 
-<p align="center">
-  <strong>boardgame.io</strong> is an engine for creating turn-based games using JavaScript.
-</p>
 
-Write simple functions that describe how the game state changes
-when a particular move is made. This is automatically converted
-into a playable game complete with online multiplayer
-features, all without requiring you to write a single line of
-networking or storage code.
+Is a groundbreaking tool that is transforming digital board game development. It smartly combines traditional game design with modern blockchain technology. This framework is designed for developers who are familiar with boardgame.io but want to improve their projects with better security and decentralization.
+At its core, T&D’s Boardgame Framework builds on boardgame.io, a popular library known for its versatility in creating various board games. What makes T&D’s Boardgame Framework unique is its integration with blockchain technology, specifically using Cartesi's capabilities.
 
-### Features
+The main feature of this framework is its ability to validate game moves using blockchain. This is achieved through two key Cartesi technologies: the Cartesi Virtual Machine (CVM) and Cartesify. 
+The CVM is an advanced system that allows complex computations to happen off-chain while keeping blockchain-level security. Cartesify, on the other hand, is a tool that simplifies the process of adapting existing applications to run in the Cartesi environment.
+
+
+
+### WiP (Work in Progress)
 
```

Comment:
It should clarify what is already implemented and what is on the roadmap. Maybe some sections should not be listed here.